docs/matchers/dialogflow.md

```diff
@@ -1,13 +1,13 @@
-# api.ai Matcher
+# Dialogflow Matcher (previously api.ai)
 
-[api.ai](https://api.ai) is an NLP API for matching strings to [intents](https://docs.api.ai/docs/concept-intents) or [actions](https://docs.api.ai/docs/concept-actions). Intents are created on the api.ai website.
+[Dialogflow](https://dialogflow.com/) is an NLP API for matching strings to [intents](https://dialogflow.com/docs/intents) or [actions](https://dialogflow.com/docs/concept-actions). Intents are created on the dialogflow website.
```

Comment:
`Dialogflow` is a proper noun and should always be capitalised in documentation (but not code).